### PR TITLE
Feature#18 new store form

### PR DESCRIPTION
**Pull request by AlanGonzalezM**
# Description

We implement the store form for crud operations.

- Now we can add stores and edit them to update the records in the database.
- We improve the visualization in both forms and tables.

## Testing

Run these commands:

`rails db:create`
`rails db:migrate`
`rails db:seed`

Start the server:

`rails serve`

## Screenshots

image

### CODE REVIEW

**Review by xManuel007 — APPROVED** [review]

**Review by MarcoRodriguez23 — APPROVED** [review]

File: app/views/employees/_employee_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+    <div class="actions">
+      <%= form.submit class: "btn btn-success btn-lg w-100"%>
+    </div>
+
+    <div class="mt-3">
+      <%= link_to "Back", employees_path,class: "btn btn-dark btn-lg w-100"%>
+    </div>
+  <% end %>
+</main>
```
Comment (juperameza) [inline app/views/employees/_employee_form.html.erb:36]: Add a new line at the end of this file
And same here with the standardized language

File: app/views/partials/_navbar.html.erb [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
     </a>
-    <% if @home %>
-      <%= link_to "Administrador", new_admin_path, class: "btn btn-success" %>
-    <% end %>
+  <% if @home %>
+    <%= link_to "Administrador", new_admin_path, class: "btn btn-success" %>
+  <% end %>
   </div>
-</nav>
+</nav>
```
Comment (juperameza) [inline app/views/partials/_navbar.html.erb:12]: New line at the end of this file

File: app/views/stores/_store_form.html.erb [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+          <%= form.submit class: "btn btn-success btn-lg w-100"%>
+        </div>
+
+        <div class="mt-3">
+          <%= link_to "Back", stores_path,class: "btn btn-dark btn-lg w-100"%>
+        </div>
+
+      </div>
+    <% end %>
+</main>
```
Comment (juperameza) [inline app/views/stores/_store_form.html.erb:24]: New line at the end of this file

**Review by juperameza — CHANGES_REQUESTED** [review]

**Review by juperameza — APPROVED** [review]
LGTM

**Pull request merged**